src/utils/createClass.js

```diff
@@ -6,10 +6,17 @@
  *  LICENSE file in the root directory of this source tree. An additional grant
  *  of patent rights can be found in the PATENTS file in the same directory.
  */
-
 export default function createClass(ctor, superClass) {
   if (superClass) {
     ctor.prototype = Object.create(superClass.prototype);
+    if (superClass.__proto__) {
+      ctor.__proto__ = superClass
```

Comment:
@tgriesser @dandelany I don't know JavaScript's prototype system well, but is it all right here? Why we check `superClass.__proto__` here but don't use it?